virtual-setup/rstudio-login.md

```diff
@@ -1 +1,55 @@
-## RStudio Server instructions
\ No newline at end of file
+## RStudio Server Set Up
+
+We will send you your Username and temporary password by a direct message on Slack, after you are logged in on [Cancer Data Science](http://ccdatalab.org/slack).
+
+Go to https://rstudio.ccdatalab.org (You may want to bookmark this) and type in your username and temporary password, and click `Sign in`.
+
+![RStudio Login](screenshots/rstudio-server-login.png)
```

Comment:
Can you use HTML for these images so you have a bit more control over the size? Some examples here: https://raw.githubusercontent.com/AlexsLemonade/training-modules/master/virtual-setup/slack-procedures.md

My rationale is that some of these important points are swamped by the unnecessarily large screenshots. I'll comment in specific cases.